### PR TITLE
luv: new package

### PR DESCRIPTION
**Pull request by milani**
Luv is libuv binding for lua.
Signed-off-by: Morteza Milani <milani@pichak.co>

Maintainer: me
Run tested: MIPS, GL-inet 6416 v1 and GL MT300A, LEDE commit dc5f496.

### CODE REVIEW

File: lang/luv/Makefile [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+PKG_NAME:=luv
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/luvit/luv.git
+PKG_SOURCE_VERSION:=1.9.1-1
+PKG_SOURCE_DATE:=2016-12-16
+
+PKG_LICENSE:=Apache License v2.0
```
Comment (karlp) [inline lang/luv/Makefile]: https://spdx.org/licenses/ says you should be using "Apache-2.0" here

File: lang/luv/Makefile [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include/luv
+	$(CP) $(PKG_INSTALL_DIR)/usr/include/luv/*.h $(1)/usr/include/luv
+	$(INSTALL_DIR) $(1)/usr/lib/
+	$(CP) $(PKG_INSTALL_DIR)/usr/lib/*.so* $(1)/usr/lib
+endef
+
+define Package/luv/install
+	$(INSTALL_DIR) $(1)/usr/lib/
```
Comment (karlp) [inline lang/luv/Makefile]: This line is redundant.

Comment (milani) [inline lang/luv/Makefile]: Fixed.

File: lang/luv/Makefile (hunk 1 above)
Comment (milani) [inline lang/luv/Makefile]: Fixed.

File: lang/luv/Makefile [hunk 3]
```diff
@@ -0,0 +1,61 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=luv
+PKG_RELEASE:=1
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=git://github.com/luvit/luv.git
```
Comment (diizzyy) [inline lang/luv/Makefile]: Use HTTPS instead

Comment (milani) [inline lang/luv/Makefile]: done

**Comment by milani** [comment]
Any chance we can get it merged?

**Comment by karlp** [comment]
Looks good to me.  :+1:

**Comment by karlp** [comment]
merged in 927ca13

**Pull request closed without merging**